### ISSUE TITLE
[Designer] Labels are not associated with the controls under "Elements Properties" pane.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6265]
### Problem Description

[30898371](https://microsoft.visualstudio.com/OS/_workitems/edit/30898371)

Screen reader is announcing only, "Checkbox (Control type) checked (value/state)'. Screen reader should announce "Initially invisible (Label name) checkbox (control type) checked (value/state)".

### Card JSON

```JSON
No card
```

**Comment by paulcam206** [comment]
This issue has been resolved in later builds (see CI site).
